Bottom: 6248b1a6ea16332eb512d3a14e0db5c3819c2e1d
Top:    e01b124173c5f52002c0dc734c02b3a450f7e098
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-25 00:48:39 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
